lib/tasks/pubmed.rake

```diff
@@ -3,6 +3,76 @@ namespace :pubmed do
     @client ||= PubmedClient.new
   end
 
+  desc 'update pubmed source records and rebuild pub_hashes for publications for a given list of cap_profile_ids passed in a text file'
+  # filename should be a list of cap_profile_id, one per line with no header
+  # call with RAILS_ENV=production bundle exec rake pubmed:refresh_source_records_for_cap_profile_ids['filename.txt']
+  task :refresh_source_records_for_cap_profile_ids, [:filename] => :environment do |_t, args|
+    filename = args[:filename]
+    fail "filename is required." unless filename.present?
+    cap_profile_ids=File.readlines(filename)
```

Comment:
I'd rather implement the incremental stuff as methods than have a rake task calling other rake tasks (which complicates logging into a single file).  And I'm not sure of the utility of a single author (which could still be run as this same rake task with just a single ID as input).  But I will implement the pubmed update into a single method.